docs/gpa.md

```diff
@@ -376,6 +378,8 @@ If you are interested in using OCDS to publish this, please contact <data@open-c
         <td markdown=1>
 
 1. [Create an OCDS release](../operations/#create-a-release)
+1. Add 'tenderUpdate' to the `tag` array
```

Comment:
At that point, have they necessarily published tender data? Is it possible they haven't published a notice of intended procurement and the related OCDS, and that they publish information about this process for the first time?

Reply:
I'm not sure such a scenario is possible: publishing tender documentation without any prior notice of the tender.

I guess there's a possibility when "turning on" their OCDS publication, that they need to refer to old processes published as non-OCDS (which for whatever reason they don't want to re-publish as OCDS), but I think that scenario is very narrow.